js/src/components/paid-ads/campaign-preview/components/placeholder.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+const styleName = {
+	thinnest: 'gla-ads-mockup__placeholder--thinnest',
+	thinner: 'gla-ads-mockup__placeholder--thinner',
+	thicker: 'gla-ads-mockup__placeholder--thicker',
+	blue: 'gla-ads-mockup__placeholder--blue',
+	'gray-100': 'gla-ads-mockup__placeholder--gray-100',
+	'gray-200': 'gla-ads-mockup__placeholder--gray-200',
+	'gray-300': 'gla-ads-mockup__placeholder--gray-300',
+	'gray-400': 'gla-ads-mockup__placeholder--gray-400',
+	'gray-500': 'gla-ads-mockup__placeholder--gray-500',
+};
+
+/**
+ * Renders a placeholder to draw a line.
+ *
+ * @param {Object} props React props.
+ * @param {boolean} [props.thinnest] Whether to draw a thinnest placeholder.
+ * @param {boolean} [props.thinner] Whether to draw a thinner placeholder.
+ * @param {boolean} [props.thicker] Whether to draw a thicker placeholder.
```

Comment:
💅  I see this just set the height of the placeholder?? 

Why not just type="thinnest|thinner|thicker" ? Or just do height={unit}  and set up the height as we do with the width.

Reply:
I forgot the original reason. I probably started with three different values of height and was inspired by the props of `Button` component (`isPrimary`, `isSecondary`, `isLink`, etc.). But when adding the color, it was not so ideal to use the same pattern, so I changed to using a single prop with different values.

And it's definitely better to have consistent props for this component. I changed them in https://github.com/woocommerce/google-listings-and-ads/pull/1650/commits/c6908de6d3f4a2abe4ee388a410ebb3884b41e39.

P.S. I have double-checked the visual results again with Figma file when doing this change. And some prop values were different from their previous value in this commit since I found them are not the closest ones to the visual design. Not sure if Figma file was changed or if I set the wrong values in the beginning. 😅